Connection/Handler.php

```diff
@@ -312,16 +317,25 @@ public function broadcastIf(\Closure $predicate, $message)
 
         $arguments = array_slice(func_get_args(), 2);
         array_unshift($arguments, $message, null);
-        $callable  = [$this, 'send'];
+        $callable   = [$this, 'send'];
+        $exceptions = new Exception\Group('Message can\'t be send to some nodes.');
```

Comment:
What do you mean by *identify nodes* ? which level ? network, socket, application ?
Here we identify exceptions by her node hash (based on the PHP ressource) : https://github.com/hoaproject/Socket/pull/36/files#diff-3b04848288f9463526ac8516775212cdR330